### PR TITLE
Add `SCMP_ARCH_NATIVE`

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

We now add the `SCMP_ARCH_NATIVE` to the seccomp profile arches.
#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Added support for seccomp CRD architecture `SCMP_ARCH_NATIVE`.
```

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]
/lgtm

File: deploy/base/crd.yaml [hunk 1]
```diff
@@ -47,27 +47,29 @@ spec:
...
+                - SCMP_ARCH_MIPSEL64N32
+                - SCMP_ARCH_PPC
+                - SCMP_ARCH_PPC64
+                - SCMP_ARCH_PPC64LE
+                - SCMP_ARCH_S390
+                - SCMP_ARCH_S390X
+                - SCMP_ARCH_PARISC
+                - SCMP_ARCH_PARISC64
+                - SCMP_ARCH_RISCV64
                 items:
```
Comment (saschagrunert) [inline deploy/base/crd.yaml:71]: This looks suspicious :thinking:

**Comment by saschagrunert** [comment]
Scope if this PR slightly changed. :D

**Review by pjbgf — APPROVED** [review]
/lgtm

**Pull request merged**